torch/onnx/symbolic_helper.py

```diff
@@ -545,7 +545,18 @@ def _is_fp(value):
                 torch.bfloat16,
             )
         else:
-            type = value.type().scalarType()
+            try:
+                type = value.type().scalarType()
+            except Exception:
+                if isinstance(value.type(), torch.ListType):
+                    type = value.type().getElementType().dtype()
```

Comment:
```suggestion
                    dtype = value.type().getElementType().dtype()
```

Avoid redefining built in names (`type`)